nielsen/src/integration/NielsenHandler.ts

```diff
@@ -24,46 +43,79 @@ export class NielsenHandler {
 
     private decoder = new TextDecoder('utf-8');
 
-    constructor(player: ChromelessPlayer, appId: string, instanceName: string, options?: NielsenOptions) {
+    constructor(
+        player: ChromelessPlayer,
+        appId: string,
+        instanceName: string,
+        options?: NielsenOptions,
+        configuration?: NielsenConfiguration
+    ) {
         this.player = player;
-        this.nSdkInstance = loadNielsenLibrary(appId, instanceName, options);
+        this.dcrEnabled = configuration?.enableDCR ?? false;
+        this.dtvrEnabled = configuration?.enableDTVR ?? true;
+        this.country = configuration?.country ?? NielsenCountry.US;
+        this.nSdkInstance = loadNielsenLibrary(appId, instanceName, options, this.country);
 
         this.addEventListeners();
     }
 
     updateMetadata(metadata: { [key: string]: string }): void {
-        this.nSdkInstance.ggPM('updateMetadata', metadata);
+        switch (this.country) {
+            case NielsenCountry.US: {
+                const { type, vidtype, assetid, ...updateableParameters } = metadata;
+                console.log(`[NIELSEN] updateMetadata: ${{ type, vidtype, assetid }} will not be updated`);
```

Comment:
Do we want to keep this log? Or was this for testing?

Reply:
I thought it made sense to warn developers about the fact that not any passed metadata can be updated mid playback. I have put it behind a check for `nol_sdkDebug === "debug"`, so it only shows if logging is enabled for the Nielsen calls also.